### DISCUSSION TITLE
Issues due to openzeppelin updated versions

### DISCUSSION THREAD

**Discussion by onkarmore26** [Cyfrin/foundry-full-course-cu#2162] [General]
In video lecture Patrick used openzeppelin version 4.8.3 in which ERC20Mock contract takes 4 parameters , but right now there is only version 5.0.2 of openzepplin is available in which ERC20Mock.sol takes only 2 parameters.

Here is the HelperConfig script code that Patrick used:
```solidity
import { ERC20Mock } from "@openzeppelin/contracts/mocks/ERC20Mock.sol";

        vm.startBroadcast();
        MockV3Aggregator ethUsdPriceFeed = new MockV3Aggregator(DECIMALS, ETH_USD_PRICE);
        ERC20Mock wethMock = new ERC20Mock("WETH", "WETH", msg.sender, 1000e8);

        MockV3Aggregator btcUsdPriceFeed = new MockV3Aggregator(DECIMALS, BTC_USD_PRICE);
        ERC20Mock wbtcMock = new ERC20Mock("WBTC", "WBTC", msg.sender, 1000e8);
        vm.stopBroadcast();
        
```

And this is the ERC20Mock.sol contract in the latest latest openzeppelin v5.0.2

```solidity
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.20;

import {ERC20} from "../../token/ERC20/ERC20.sol";

contract ERC20Mock is ERC20 {
    constructor() ERC20("ERC20Mock", "E20M") {}

    function mint(address account, uint256 amount) external {
        _mint(account, amount);
    }

    function burn(address account, uint256 amount) external {
        _burn(account, amount);
    }
}
```

Can someone help me figure out how i can resolve this issue ?

**Comment by EngrPips** [comment]
Hello @onkarmore26, This was addressed in the `chronological Update` section of this GitHub https://github.com/Cyfrin/foundry-full-course-cu/blob/main/chronological-updates.md . Whenever you notice something has changed, you should visit the `Chronological Update` section of this Github to see if it has been addressed.

**Reply by onkarmore26** [reply]
I got it now. Thanks alot for your assistance

**Reply by EngrPips** [reply]
You are welcome.

**Reply by ervand7** [reply]
Thank you, it works. I just leave it here:
forge install openzeppelin/openzeppelin-contracts@v4.8.3 --no-commit

**Reply by EngrPips** [reply]
You are welcome, keep crushing it.
